### PR TITLE
Typography Fine-Tuning

### PR DESCRIPTION
**Pull request by MariaNono**
remove unnecessary font-faces (with font-weights 200 and 300)
add sans-serif to font-family declaration
fine-tune letter spacing and font-weights

### CODE REVIEW

File: _sass/components/_section.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		p {
 			font-family: _font(family-heading);
 			font-size: 1em;
-			font-weight: _font(weight-heading-alt);
+			font-weight: _font(weight);
 			letter-spacing: _font(kerning-heading);
```
Comment (Smokyish) [inline _sass/components/_section.scss:32]: `0.03em` is not the right `letter-spacing` we should be using, it should be not declared for `p`.

File: _sass/components/_button.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@
 		cursor: pointer;
 		display: inline-block;
 		font-size: 1em;
-		font-weight: _font(weight-heading);
+		font-weight: _font(weight-bold);
 		height: 2.5em;
 		letter-spacing: _font(kerning-heading);
```
Comment (Smokyish) [inline _sass/components/_button.scss:22]: `0.03em` is not the right letter-spacing we should be using, it should be not declared for `button`.

**Review by Smokyish — CHANGES_REQUESTED** [review]
Go through and remove all unnecessary `letter-spacing` declarations, i.e., use of `letter-spacing: _font(kerning-heading);` and correct the `letter-spacing: -1px;` to `-0.5px;`.

Also see individual comments in the review.

Please resolve the merge conflicts as well.

File: _sass/base/_typography.scss [hunk 3]
```diff
@@ -76,10 +75,13 @@ h1 {
 	font-size: 3em;
 	margin: 0 0 (_size(element-margin) * 0.25) 0;
 	line-height: 1.2;
+	letter-spacing: -1px;
```
[the remark this answers is not in the document]
Comment (MariaNono) [reply _sass/base/_typography.scss:78]: -1px looks the best and is what is specified on the guidelines.
Screenshot 2020-09-28 at 13 10 11

File: _sass/components/_section.scss (hunk 1 above)
Comment (MariaNono) [reply _sass/components/_section.scss:32]: I will remove it but these rules only apply to "section header p", _p_ inside _header_ inside _section_, and that never happens anywhere. Probably the whole block should be removed?

**Review by Smokyish — APPROVED** [review]
All good now.

**Pull request merged**